tests/runci/targets/Php.hx

```diff
@@ -49,25 +46,43 @@ class Php {
 		final binDir = "bin/php";
 
 		final prefixes = [[]];
-		if(isCi()) {
+		if (isCi()) {
 			prefixes.push(['-D', 'php-prefix=haxe']);
 			prefixes.push(['-D', 'php-prefix=my.pack']);
 		}
 
-		for(prefix in prefixes) {
+		for (prefix in prefixes) {
 			changeDirectory(unitDir);
-			if(isCi())
+			if (isCi())
 				deleteDirectoryRecursively(binDir);
 
 			runCommand("haxe", ["compile-php.hxml"].concat(prefix).concat(args));
 			runCommand("php", [binDir + "/index.php"]);
 
 			changeDirectory(sysDir);
-			if(isCi())
+			if (isCi())
 				deleteDirectoryRecursively(binDir);
 
 			runCommand("haxe", ["compile-php.hxml"].concat(prefix).concat(args));
-			runSysTest("php", ["bin/php/Main/index.php"]);
+			runThroughPhpVersions(runSysTest.bind(_, ["bin/php/Main/index.php"]));
+
+			changeDirectory(asysDir);
+			if (isCi()) {
+				deleteDirectoryRecursively(binDir);
+			}
+			runCommand("haxe", ["compile-php.hxml"].concat(prefix).concat(args));
+			runThroughPhpVersions(runCommand.bind(_, ['$binDir/index.php']));
+		}
+	}
+
+	static function runThroughPhpVersions(fn:(phpCmd:String) -> Void) {
+		switch [ci, systemName] {
+			case [GithubActions, "Linux"]:
+				for (version in ['7.4', '8.0']) {
+					fn('php$version');
+				}
+			case _:
+				fn('php');
```

Comment:
@Simn If you merge from development once again it should show the conflict. This is how it should be resolved:
```suggestion
			runSysTest("php", generateArgs(binDir + "/Main/index.php"));

			changeDirectory(asysDir);
			if (isCi()) {
				deleteDirectoryRecursively(binDir);
			}
			runCommand("haxe", ["compile-php.hxml"].concat(prefix).concat(args));
			runCommand("php", generateArgs(binDir + "/index.php"));
```